cves/CVE-2015-6826.yml

```diff
@@ -186,8 +212,13 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    Since the vulnerability was exploitable by a specifically crafted file,
+    it's reasonable to assume that a fuzzer could have discovered the file.
+
+    Also, given that multiple fields in a struct were left uninitialized,
+    compiler warnings may have revealed them if configured to do so.
```

Comment:
While I agree with the compiler warnings point, wasn't your previous CVE also only viable via a crafted file and discovered by a fuzzer?

Reply:
On second look, I misread your comment regarding the fuzzer - apologies for that confusion

Reply:
No problem!